.werft/platform-trigger-artificial-job.sh

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env bash
+#
+# The easiest way to run this script is through Werft so you don't have to worry
+# about installing the appropraite service account etc. locally.
+#
+#   werft job run github -j .werft/platform-trigger-artificial-job.yaml -s .werft/platform-trigger-artificial-job.sh
+#
+
+sleep 1
+
+set -Eeuo pipefail
+
+# Activate service account and install core-dev context
+gcloud auth activate-service-account --key-file "/mnt/secrets/gcp-sa/service-account.json"
+gcloud container clusters get-credentials core-dev --zone europe-west1-b --project gitpod-core-dev
+
+echo "[Creating new branch|PHASE] Trigger new job with preview environment"
+echo "[Creating new branch] Setting up user config"
+git config --global user.name roboquat
+git config --global user.email roboquat@gitpod.io
+git remote set-url origin https://oauth2:"${ROBOQUAT_TOKEN}"@github.com/gitpod-io/gitpod.git
+
+echo "[Creating new branch] Checking out branch"
+git checkout -b "artificial-job-$(echo $RANDOM | md5sum | head -c 20; echo;)"
+echo "[Creating new branch] Creating commit"
+git commit -m "Creating an artificial traffic for the Preview Environment start SLO" --allow-empty
+echo "[Creating new branch|DONE] Trigger new job with preview environment"
```

Comment:
Could be `echo "Trigger new job with preview environment" | werft log slice "Creating new branch" --done`